src/main/java/com/chimbori/crux/articles/PostprocessHelpers.java

```diff
@@ -203,4 +208,13 @@ private static void removeDisallowedAttributes(Element node) {
     }
   }
 
+  private static boolean isImportant(Node node) {
```

Comment:
The method name could be a bit clearer about how the node was deemed important, and what importance really means (It could also imply that this is an `h1` for example). How about `shouldKeep`, and name the attribute `crux-keep`?

Reply:
changed!